mobile/src/pods/InviteUsers.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { useState } from "react";
+import {
+  View,
+  Text,
+  FlatList,
+  ActivityIndicator,
+  SafeAreaView,
+  StatusBar,
+  StyleSheet,
+  Button,
+} from "react-native";
+import { List, ListItem, SearchBar } from "react-native-elements";
+import { useSelector } from "react-redux";
+
+interface User {
+  id: number;
+  email: string;
+}
+
+const InviteUsers: React.FC<{}> = ({ navigation, route }) => {
+  const u: User[] = [];
+  const [users, setUsers] = useState(u);
+  const [invitees, setInvitees] = useState([]);
+
+  const currUserId = useSelector((state) => state.auth.user.id);
+
+  React.useEffect(() => {
+    async function fetcher() {
+      try {
+        const res = await fetch("http://localhost:8000/users", {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json;charset=utf-8",
+          },
+        });
+
+        const json = await res.json();
+        const result = json.filter((user) => user.id != currUserId);
+        setUsers(result);
+      } catch (err) {
+        console.log("error loading users");
+      }
+    }
+
+    fetcher();
+  }, []);
+
+  const Item = ({ title, user }) => (
```

Comment:
Maybe this could use a more descriptive name and could be moved into its own file?

Reply:
wdyt?

Reply:
Yeah I could use a more descriptive name for sure. I don't really think it needs to go into its own file since it won't be used anywhere but here